_run/common-kind.mk

```diff
@@ -1,19 +1,41 @@
-KIND_NAME      ?= $(shell basename $$PWD)
+KIND_NAME      = kind
+# 'kind' matches the configuration of the GH actions KinD. 
 K8S_CONTEXT    ?= $(shell kubectl config current-context)
-KIND_HTTP_PORT  = $(shell docker inspect \
+APP_HTTP_PORT  ?= $(shell docker inspect \
```

Comment:
why rename this one to `APP_` when the others are `KIND_`?  It is used elsewhere.

Reply:
good catch.